charts/rmd/templates/cronjob.yaml

```
@@ -27,7 +27,11 @@ spec:
                 {{- include "rmd.djangoDatabasePassword" $ | nindent 16 }}
               envFrom:
                 - secretRef:
-                    name: {{ include "rmd.fullname" $ }}
+                    {{- if $.Values.rmd.existingSecret}}
+                    name: {{ $.Values.rmd.existingSecret}}
+                    {{- else }}
+                    name: {{ include "rmd.fullname" . }}
```

it looks like the `rmd.fullname` funciton is getting passed the wrong context.
```suggestion
                    name: {{ include "rmd.fullname" $ }}
```

`rmd.fullname` expects a top-level context and `.` doesn't contain since we are in a range.